config/database.yml

```diff
@@ -16,6 +16,8 @@
 #
 default: &default
   adapter: postgresql
+  username: postgres
+  password: 96939066
```

Comment:
remove these two lines or at least mention in the readme file that the users should use their corresponding username and password.